apps/challenger/src/app.mjs

```diff
@@ -53,6 +33,8 @@ app.use(
 );
 
 app.get('/healthcheck', (req, res) => res.sendStatus(200));
+if (!CHALLENGER_PORT) throw new Error('Please specify a challenger port');
```

Comment:
Will this exception ever happen? On line `12` we specify a fallback value of `8192` in case the port is not defined. Either we should get rid of the default value from line 12, or remove this condition.

Reply:
you're right. Thanks